plugins/broker-mqtt/src/static/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface MqttMessage {
+  key: string | Record<string, unknown> | null;
+  value: string | Record<string, unknown> | null;
+  headers: MqttMessageHeaders;
+}
```

Comment:
`message: MqttMessage` is a bit too strict for mqtt. I'd suggest to refactor the type here to something that will allow either a simple value without `key` and `headers`.
i.e. 
```ts
export type MqttMessage = string | Record<string, unknown> | null;
```

Reply:
Sure.